src/app/modules/group/components/group-log-view/group-log-view.component.ts

```diff
@@ -130,27 +118,17 @@ export class GroupLogViewComponent implements OnChanges, OnInit, OnDestroy {
 
     const target = event.target;
     const relatedTarget = event.relatedTarget;
-    let closeOverlay = true;
-
-    if (target instanceof HTMLElement && relatedTarget instanceof HTMLElement) {
-      let currentElement = relatedTarget;
-      while (currentElement.parentElement) {
-        if (currentElement.classList.contains('alg-path-suggestion-overlay')) {
-          closeOverlay = false;
-          break;
-        }
-        currentElement = currentElement.parentElement;
-      }
-    }
```

Comment:
I tested it and it looked to me to be exactly equivalent to [`.closest(selector)` function](https://developer.mozilla.org/en-US/docs/Web/API/Element/closest).